cibyl/sources/jenkins.py

```diff
@@ -116,6 +116,17 @@ def has_filter_builds(**kwargs):
     return any(arg.value for arg in build_args.values())
 
 
+def has_filter_tests(**kwargs):
```

Comment:
Something not important but I realized some places of the code has the type in the parameters and also the return type established and other places no. Can we standardize it? (We can talk about it in general)

Reply:
That's a good point, I try to add them since they help with readability, but most of the time I forget. I'll try to do better :)